specs/engage-digital_openapi3.yaml

```diff
@@ -938,6 +938,12 @@ paths:
           schema:
             format: int32
             type: integer
+        - description: Filter by type
+          in: query
+          name: type
+          required: false
+          schema:
+            type: string

```

Comment:
We can probably add an `enum` with the list of accepted types (like what we do for permissions).